Scripts/RemediationScripts/Remediate-DeleteNonAADIdentityProvidersInAPIManagementServices.ps1

```diff
@@ -1,13 +1,13 @@
 ﻿<###
 # Overview:
-    This script is used to delete all non-Azure Active Directory (AAD) identity providers from all API Management services in a Subscription.
+    This script is used to delete all non-Azure Active Directory (Entra Id) identity providers from all API Management services in a Subscription.
```

Comment:
AAD should be in brackets, and be consistent with how you're using Entra ID,
1. Microsoft Entra ID
2. Microsott Entra Id
3. Entra ID
4. Entra Id

Need to pick one

Reply:
for control Ids - Mircosoft_Entra_ID is being used,
all other places Entra Id is used.